include/controller/controller.h

```diff
@@ -18,20 +18,9 @@ enum {
     ACT_FLUSH = 0x100
 };
 
-enum {
-    CTRL_TYPE_KEYBOARD,
-    CTRL_TYPE_GAMEPAD,
-    CTRL_TYPE_NETWORK,
-    CTRL_TYPE_AI,
-    CTRL_TYPE_REC
-};
+enum { CTRL_TYPE_KEYBOARD, CTRL_TYPE_GAMEPAD, CTRL_TYPE_NETWORK, CTRL_TYPE_AI, CTRL_TYPE_REC };
 
-enum {
-    EVENT_TYPE_ACTION,
-    EVENT_TYPE_SYNC,
-    EVENT_TYPE_HB,
-    EVENT_TYPE_CLOSE
-};
+enum { EVENT_TYPE_ACTION, EVENT_TYPE_SYNC, EVENT_TYPE_HB, EVENT_TYPE_CLOSE };
```

Comment:
let's try to find a setting where enums are on separate lines

Reply:
maybe `AllowShortEnumsOnASingleLine: false` could work

Reply:
Looks like `AllowShortEnumsOnASingleLine: false` works, but it puts the starting brace on a new line, and doesn't look like
```
BraceWrapping:
  AfterEnum:       false
```
is enough to put the brace on same as the enum ..

Reply:
Probably this: https://reviews.llvm.org/D93938'

Let's just leave it as true.

Reply:
So the current configuration will now fix this issue when clang-format gets fixed.

Reply:
No, I didn't want to leave any bombs to the config that would cause odd behaviour on version update. Let's just manually set it to false and update the clang-format when new one is released.